src/components/OptionsList/BaseOptionsList.tsx

```diff
@@ -138,13 +138,13 @@ function BaseOptionsList(
      */
     // eslint-disable-next-line @typescript-eslint/naming-convention
     const getItemLayout = (_data: OptionsListDataWithIndexOffset[] | null, flatDataArrayIndex: number) => {
-        if (!flattenedData.current[flatDataArrayIndex]) {
+        if (!flattenedData.current.at(flatDataArrayIndex)) {
```

Comment:
Please make sure `flatDataArrayIndex` won't be negative since it can affect the logic then.